support/test-api/src/main/java/com/hotels/styx/testapi/StyxServer.java

```diff
@@ -208,4 +248,16 @@ public StyxServer start() {
             return new StyxServer(this).start();
         }
     }
+
+    private static class PluginTriple {
```

Comment:
The name `Triple` describes the structure of the class, but the name should describe its purpose. The structure is clear from the code.

Reply:
Maybe `PluginConfiguration` or `PluginFactoryConfiguration`.

Reply:
fixed